decidim-comments/app/types/decidim/comments/commentable_interface.rb

```diff
@@ -49,6 +49,12 @@ module Comments
           obj.comment_threads.size.positive?
         }
       end
+
+      field :userCanComment, !types.Boolean, "Check if the current user can comment" do
+        resolve lambda { |obj, _args, ctx|
```

Comment:
@isaacmg410 here are `obj` and `ctx`, use those!

Reply:
@mrcasals I have created this field. The problem is that I overwrite the method `user_can_comment?(user)`  for example `meetings`, and I try to pass the user without success... And comments doesn't load...

Reply:
Can't you just do something like?

`obj.commentable? && obj.can_participate?(ctx[:current_user])`

Reply:
This is what I have already solved, in another branch. I will create a new PR today and close this one.

The question now is that I do not have any idea on how to solve and render the comment to the public side. Any idea?

Reply:
> The question now is that I do not have any idea on how to solve and render the comment to the public side. Any idea?

I don't understand this, aren't the comments already rendered?